ui/src/state/PatronCurrentBorrowingRequests/actions.js

```diff
@@ -0,0 +1,44 @@
+import { IS_LOADING, SUCCESS, HAS_ERROR } from './types';
+import { invenioConfig } from '@config';
+import { illBorrowingRequest as BorrowingRequestApi } from '@api';
+import { sendErrorNotification } from '@components/Notifications';
+
+const selectQuery = (patronPid, page = 1, size) => {
+  const statuses = invenioConfig.illBorrowingRequests.pendingStatuses.concat(
+    invenioConfig.illBorrowingRequests.requestedStatuses.concat(
+      invenioConfig.illBorrowingRequests.activeStatuses
+    )
+  );
+  return BorrowingRequestApi.query()
+    .withPatron(patronPid)
+    .withState(statuses)
+    .withSize(size)
+    .withPage(page)
+    .qs();
+};
+
+export const fetchPatronCurrentBorrowingRequests = (
+  patronPid,
+  page,
+  size = invenioConfig.defaultResultsSize
+) => {
+  return async dispatch => {
+    dispatch({
+      type: IS_LOADING,
+    });
+    await BorrowingRequestApi.list(selectQuery(patronPid, page, size))
+      .then(response => {
```

Comment:
can you please use the `async/await` form instead? See my PR for example [here](https://github.com/inveniosoftware/invenio-app-ils/pull/827/files#diff-64662b066580e283b8170b72d12edd61R8)